task.go

```diff
@@ -503,6 +503,8 @@ func (t *task) Checkpoint(ctx context.Context, opts ...CheckpointTaskOpts) (Imag
 type UpdateTaskInfo struct {
 	// Resources updates a tasks resource constraints
 	Resources interface{}
+	// Annotations allows for experimentation with updating resource constraints
```

Comment:
Only for experimentation?

Reply:
I can update this. Really annotations are for arbitrary AND experimental resource constraint information. The goal is that we can pass in any resource information for use and for testing now, then if we determine there are specific resources or other fields that would be useful upstream or in some other way, we can make additional PRs to opencontainer's runtime spec to update WindowsResources.